File: app/src/main/java/com/ddancn/keepaccount/adapter/TypeAdapter.java
package com.ddancn.keepaccount.adapter;

import androidx.annotation.Nullable;

import com.chad.library.adapter.base.BaseQuickAdapter;
import com.chad.library.adapter.base.BaseViewHolder;
import com.ddancn.keepaccount.R;
import com.ddancn.keepaccount.entity.Type;

import java.util.List;

/**
 * @author ddan.zhuang
 */
public class TypeAdapter extends BaseQuickAdapter<Type, BaseViewHolder> {

    public TypeAdapter(int layoutResId) {
        super(layoutResId);
    }

    public TypeAdapter(int layoutResId, @Nullable List<Type> data) {
        super(layoutResId, data);
    }

    @Override
    protected void convert(BaseViewHolder helper, Type item) {
        helper.setText(R.id.tv_type, item.getName());
    }

}

File: app/src/main/java/com/ddancn/keepaccount/adapter/TypeAdapter.kt
package com.ddancn.keepaccount.adapter

import com.chad.library.adapter.base.BaseQuickAdapter
import com.chad.library.adapter.base.BaseViewHolder
import com.ddancn.keepaccount.R
import com.ddancn.keepaccount.entity.Type

/**
 * @author ddan.zhuang
 */
class TypeAdapter(layoutResId: Int, data: List<Type>? = null)
    : BaseQuickAdapter<Type, BaseViewHolder>(layoutResId, data) {

    override fun convert(helper: BaseViewHolder, item: Type) {
        helper.setText(R.id.tv_type, item.name)
    }

}
